tools/Runner.py

```diff
@@ -0,0 +1,6 @@
+class Runner:
```

Comment:
We shouldn't have a `Runner.py` and a `runner` python script in the same directory. We need to organize  it better.

Reply:
I've renamed `Runner.py` to `BaseRunner.py` to at least make filenames less confusing